appendix/backup-and-restore/index.rst

```diff
@@ -0,0 +1,86 @@
+Backup and Restore
+******************
+
+Zammad comes with a collection of scripts for easy backup & restore for default
+installations. These scripts are located within ``/opt/zammad/contrib/backup``.
+
+.. note:: **⚖️ Important things to note before hand**
+
+   These scripts do not come with any warranty and may not work in your
+   specific use case. This depends on the configuration and installation type
+   of your instance.
+
+   You should always regularly test and review the functionality!
+   If the script functionality or scope is not working for your cases,
+   feel free to copy these to a independent location and adjust the scripts
+   as needed.
+
+Getting Started
+===============
+
+.. toctree::
+   :hidden:
+
+   configuration
+   backup
+   restore
+   migrate-hosts
+   troubleshooting
+   little-helpers
+
+Before you continue, please also note the listed limitations to save your
+precious time.
+
+   :doc:`🔨 Adjust script settings </appendix/backup-and-restore/configuration>`
+      Learn more about configuration options for backup and restore to see
+      scopes better.
+
+   :doc:`🗃️ Create Backups </appendix/backup-and-restore/backup>`
+      How to create full dumps of your Zammad installation.
+
+   :doc:`🗄️ Restore Backups </appendix/backup-and-restore/restore>`
+      Update went berserk and you need to go back? How to restore your instance on
+      a new or the same host.
+
+   :doc:`🔀 Migrating to new hosts </appendix/backup-and-restore/migrate-hosts>`
+      This is a general summary on how to best migrate Zammad from host to host.
+      We'll reference to backup creation and restoration as needed.
+
+   :doc:`🔥 Troubleshooting </appendix/backup-and-restore/troubleshooting>`
+      Things hit the fan? This page might help you out of that pit.
+
+   :doc:`🤝 Helper scripts </appendix/backup-and-restore/little-helpers>`
+      These scripts may be helpful if Backup & Restore does not work as expected.
+      However note that these are potentially destructive.
+
+Limitations
+===========
+
+Please note the following limitations which may affect script functionality
+or availability.
+
+   * Restoration via script on docker based installations may not work and
```

Comment:
IMO we should add source code installations too. Because it can be a very different environment as the backup/restore script expects. Just my 2 cents.